sdk/monitor/Azure.Monitor.Query/tests/MetricsQueryClientTests.cs

```diff
@@ -60,5 +63,54 @@ public void ExposesPublicEndpoint()
             var client = new MetricsQueryClient(new Uri("https://management.azure.gov"), new MockCredential(), new MetricsQueryClientOptions());
             Assert.AreEqual(new Uri("https://management.azure.gov"), client.Endpoint);
         }
+
+        [Test]
+        public void CanGetMetricQueryResult()
+        {
+            var metadata = new Dictionary<string, string> { { "metadatatest1", "metadatatest2" } };
+            var metricValue = new MetricValue(new DateTimeOffset(new DateTime(10)));
```

Comment:
Is this ctor public?

Reply:
sorry, yes I kept updating the Generated file by mistake. I believe I modified the correct one now.